composer.json

```
@@ -31,7 +31,7 @@
         }
     },
     "require": {
-        "php": "^7.4 || ^8.0 || ^8.1",
+        "php": "^7.4 || ~8.0 || ~8.1.0",
```

`~1.0` is equivalent to `^1.0` or `1.*`, but `~1.0.0` is equivalent to `1.0.*`.

```suggestion
        "php": "^7.4 || ~8.0.0 || ~8.1.0",
```